plugins/jira-dashboard-backend/src/api.ts

```diff
@@ -93,8 +93,6 @@ export type SearchOptions = {
  * @param config - A Backstage config
  * @param jqlQuery - A string containing the jql query.
  * @param options - Query options that will be passed on to the POST request.
- *
- * @public
```

Comment:
We need to add the "@public" here? The "searchJira" is not included in the "api-report"?

Reply:
Right... I just aded it back.